incident_scraper/external/google_nbd.py

```diff
@@ -28,6 +28,7 @@ def __init__(self):
     def _create_incident_from_dict(incident: dict):
         """Convert an incident dict to a Incident Model."""
         return Incident(
+            id=incident["UCPD_ID"],
```

Comment:
Made the `UCPD ID` field the ID, so that we do not get copies of the same report in the Datastore.